ansible/library/port_alias.py

```diff
@@ -98,6 +100,11 @@ def get_portmap(self, asic_id=None, include_internal=False):
         front_panel_asic_ifnames = []
         # All asic names
         asic_if_names = []
+        sysports = []
+        port_coreid_index = -1
+        port_core_portid_index = -1
+        num_voq_index = -1
+        asic_name = "ASIC0" if asic_id is None else "ASIC" + str(asic_id)
```

Comment:
What is the reason for having a default asic name as "ASIC0" ? Can this be empty string instead?

Reply:
https://github.com/Azure/sonic-buildimage/pull/5991/files#diff-466ae5de99730b4f3dde9c932e9086536e3a0226858c735fbf776453e8d86fafR1292

has this default assumption of Asic0 in minigraph.py
       results['DEVICE_METADATA']['localhost']['asic_name'] = 'Asic0'

Reply:
thank you, might be good to add that comment here as done in minigraph parser.